### DISCUSSION TITLE
Windows Speech Recognition (WSR) deprecation and System.Speech.Recognition

### DISCUSSION THREAD

**Discussion by opus49** [dotnet/runtime#102317] [Q&A]
Will the deprecation and potential removal of Windows Speech Recognition impact the System.Speech.Recognition library?

**Comment by huoyaoyuan** [comment]
System.Speech is just a wrapper of Windows Speech Recognition. It should be affected by the availability of WSR.

**Reply by opus49** [reply]
Has Microsoft said anything about making Voice Access accessible from .NET via an API?

**Reply by huoyaoyuan** [reply]
I don't know. You can always create a wrapper with P/Invoke and COM interop yourself.
